repository/l.json

```diff
@@ -1604,6 +1604,17 @@
 				}
 			]
 		},
+		{
+			"name": "Lotus Color Scheme",
+			"details": "https://github.com/luxelego/lotus_color_scheme",
+			"labels": ["color scheme"],
+			"releases": [
+				{
+					"sublime_text": "*",
+					"branch": "master"
```

Comment:
Please change this to "tags": true so that package control will pick up your tags, not simply every change to master (which is deprecated anyway)